src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -722,6 +724,34 @@ private void verifyPoWithPoLinesUpdate(JsonObject reqData, JsonObject storageDat
     assertEquals(MockServer.serverRqRs.get(PO_LINES, HttpMethod.PUT).size(), sameLinesCount);
   }
 
+  @Test
+  public void testUpdateOrderWithStatus_MODORDERS150() throws Exception {
+    logger.info("=== Test Put Order By Id - MODORDERS-150 ===");
```

Comment:
A description of the test would be easier to debug later here instead of the JIRA ticket number.

Reply:
The jira reference replaced by description.